client/src/components/Menu/Menu.scss

```diff
@@ -771,6 +771,10 @@
       .cms-help__badge {
         display: block;
       }
+
+      .cms-help__logo {
+        flex-shrink: 0;
```

Comment:
Put this on line 693 with the rest of logo rules.

Reply:
updated